Tictactoe.cs

```diff
@@ -134,10 +134,7 @@ private bool MakeMove(string input)
 
         private bool CanTakeTurn(string input)
         {
-            var hasUserQuit = UserHasQuit(input);
-            var isInputValid = Board.IsValidCoordinate(input);
-
-            return isInputValid && !hasUserQuit;
+            return Board.IsValidCoordinate(input);
```

Comment:
It looks like you don't need this method now and can just inline `Board.IsValidCoordinate` in `LetPlayerMakeMove`